src/index.ts

```diff
@@ -101,38 +202,49 @@ export default class NestedList {
    * @returns {Element}
    * @public
    */
-  render() {
-    this.nodes.wrapper = this.makeListWrapper(this.data.style, [ this.CSS.baseBlock ]);
+  render(): Element {
```

Comment:
rendering is not working for some reason


![image](https://github.com/user-attachments/assets/f35e2e1a-9169-4c2c-8596-4a32cf02ae6f)

Reply:
And the conversion to nested list from another block does not work as well

Reply:
You can test it by connecting the tool locally to the Editor.js dev-server page. Now it [uses](https://github.com/codex-team/editor.js/blob/next/index.html#L84) the version from CDN. You can remove CDN script connection and connect the locally build version (which is available as a submodule, or anyone on you local machine).

Example

<img width="768" alt="image" src="https://github.com/user-attachments/assets/c922645d-52fd-4de3-b9eb-55a5d6ebe73c">

Then 
1. Build nested list
2. Run `yarn dev` in editor.js folder

Reply:
i found the problem, its the code i changed to avoid TS error of setting read-only attributes, now i change back to how it was used to set attribute.
 ```
 type HtmlElementAttributes = Partial<HTMLElement>;
 
 // this is the problem code, the el.setAttribute does not set innerHTML property
 // if (attributes) {
  //   Object.entries(attributes).forEach(([key, value]) => {
  //     if (!value) return;
  //     el.setAttribute(key, value.toString());
  //   });
  // }

  for (const attrName in attributes) {
    // as any is used to avoid TS error that read-only properties of HTMLElement are not assignable
    (el[attrName as keyof HtmlElementAttributes] as any) =
      attributes[attrName as keyof HtmlElementAttributes];
  }
```